### PR TITLE
fix(auto-place): complete direct editing on autoPlace init

### PR DESCRIPTION
**Pull request by pinussilvestrus**
# BREAKING CHANGES

* `autoPlace` event renamed to `autoPlace.getPosition`

Related to camunda/camunda-modeler#1502

Kapture 2019-09-11 at 11 30 55

### CODE REVIEW

**Comment by philippfromme** [comment]
Make sure to update the event name in all the places where we've been using it (e.g. [`AutoPlaceBehavior`](https://github.com/bpmn-io/bpmn-js/blob/develop/lib/features/grid-snapping/behavior/AutoPlaceBehavior.js#L8).

**Comment by pinussilvestrus** [comment]
This is actually the only place were we are using the original `autoplace` event [for providing the autoplace position](https://github.com/bpmn-io/bpmn-js/blob/develop/lib/features/auto-place/AutoPlace.js#L29). All other occurrences uses the `autoplace.end` event. We could consider renaming this to something like `autoplace.providePosition` to make it more clear.

File: lib/features/auto-place/AutoPlace.js [hunk 1]
```diff
@@ -25,6 +25,12 @@ export default function AutoPlace(eventBus, modeling) {
    */
   this.append = function(source, shape) {
 
+    // notify interested parties on auto place started
```
Comment (philippfromme) [inline lib/features/auto-place/AutoPlace.js]: We're basically firing the same event twice.

File: test/spec/features/auto-place/AutoPlaceSpec.js [hunk 2]
```diff
@@ -230,6 +254,67 @@ describe('features/auto-place', function() {
...
+    it('should emit <init>', inject(
+      function(autoPlace, elementRegistry, elementFactory, eventBus) {
+
+        // given
+        var el = elementFactory.createShape({ type: 'bpmn:Task' });
+
+        var source = elementRegistry.get('TASK_2');
+
+        var listener = sinon.spy(function(event) {
+          expect(event.source).to.eql(source);
```
Comment (philippfromme) [inline test/spec/features/auto-place/AutoPlaceSpec.js]: We're interested in `equal` here.

File: lib/features/auto-place/AutoPlace.js (hunk 1 above)
Comment (pinussilvestrus) [reply lib/features/auto-place/AutoPlace.js]: Not Sure. If we listen to autoPlace.end, the directEditing got still activated for the newShape, so we won't be able to complete it for the source in the LabelEditingProvider

Comment (pinussilvestrus) [reply lib/features/auto-place/AutoPlace.js]: We should complete it just _before_ the new shape was created.

**Comment by nikku** [comment]
Quick feedback:

* A fix can _NEVER_ ship with breaking changes, please revert the breaking change on master, if you decide to publish it as a patch release.

We can decide to break things but please don't mark it as a fix then.

**Comment by philippfromme** [comment]
> * A fix can _NEVER_ ship with breaking changes, please revert the breaking change on master, if you decide to publish it as a patch release

Makes sense. I'll remove the breaking change for now so it becomes a fix. Will be more careful in the future. 👍

**Pull request merged**